redirect_rules.py

```diff
@@ -571,6 +584,5 @@
     print("\n[+]\tTotal IPs, Networks or User-Agents blocked: %d" % result)
     print("[+]\tRedirect rules file: %s" % WORKINGFILE_NAME)
 
-
     elapsed = time.perf_counter() - start
-    print(f"\n{__file__} executed in {elapsed:0.2f} seconds.")
+    print("\n{__file__} executed in {elapsed:0.2f} seconds.")
```

Comment:
Was there a reason for removing the f-string? This change causes the final execution timer displayed to show the explicit string above instead of the file name and elapsed time.

Reply:
Ah good catch, I'll add the `f` back